docs/sources/query/log_queries/_index.md

```diff
@@ -463,6 +463,27 @@ The **logfmt** parser can operate in two modes:
     "fwd_ip" => "124.133.124.161"
     ```
 
+logfmt parser also supports `--strict` flag to enable strict parsing.
+
+```
+| logfmt --strict
+| logfmt --strict host, fwd_ip="fwd"
+```
+
+With strict parsing enabled, logfmt parser stops scanning the log line and returns early with an error if it encounters any poorly formatted key=value pair.
+```
+// accepted
+key=value key="value in double quotes" standalone_key
+
+// invalid
+=value // no key
+foo=bar=buzz
+fo"o=bar
+```
+
+By default, the parser skips the invalid tokens and continues parsing the rest of the log line.
+This offers the flexibility to parse semi-structed log lines, though note that this is only best-effort and the resulting label set might not always contain what you expect.
```

Comment:
"This" is unclear, I'm not sure what you're referring to (the default behavior?)